Tasks/Maven/maventask.ts

```diff
@@ -282,9 +306,15 @@ function applyPmdGoals(mvnRun: trm.ToolRunner):void {
     pmd.applyPmdArgs(mvnb);
 }
 
-// Returns true if any one code analysis tool was enabled.
-function isCodeAnalysisUploadRequired():boolean {
-    return tl.getInput('pmdAnalysisEnabled', true) == 'true';
+// Returns the names of any enabled code analysis tools, or empty array if none.
+function getEnabledCodeAnalysisTools():string[] {
```

Comment:
This function was changed to provide an iterable list of enabled code analysis tools, as well as fulfilling its original function.